server/superdesk/io/rfc822.py

```diff
@@ -54,11 +72,18 @@ def parse_email(self, data, provider):
                 if isinstance(response_part, tuple):
                     msg = email.message_from_bytes(response_part[1])
                     item['headline'] = self.parse_header(msg['subject'])
-                    item['original_creator'] = self.parse_header(msg['from'])
+                    field_from = self.parse_header(msg['from'])
+                    item['original_source'] = field_from
```

Comment:
Although, this persists successfully in Mongo this will fail when the call is routed through eve. Please register this property in `content.py` module to avoid this.